guardian/mobile-save-for-later
# Upgrade to java11

## What does this change?

This upgrades the lambdas that serve the Save For Later functionality from Java8 to Java11

## How to test
Tested Save for Later on CODE:

- [x] Can get articles
- [x] Can post articles

Tested User Deletion CODE:

- [x] Delete a user account and see that this is deleted from Save For Later

Note I wasn't able to test the full interaction on CODE as I have been unable to find the exact string the app is expecting to parse. I think the json parsing logic needs a bit of an update too, so I'll make a note to do that and this should make it possible to test an example event. 

For now, we have alarms set up on PROD if the delete interaction is unable to complete. 


## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
